wgpu-core/src/conv.rs

```diff
@@ -755,3 +755,16 @@ pub fn map_index_format(index_format: wgt::IndexFormat) -> hal::IndexType {
         wgt::IndexFormat::Uint32 => hal::IndexType::U32,
     }
 }
+
+/// Take `value` and round it up to the nearest alignment `alignment`.
+///
+/// ```text
+/// (0, 3) -> 0
```

Comment:
should this be written as Rust assertions?

Reply:
I tried, but because the conv module isn't visible from the outside, and doctests are technically completely different crates, I couldn't actually call the function in the doctest.